lib/configproxy.js

```diff
@@ -174,6 +175,7 @@ ConfigurableProxy.prototype.add_route = function (path, data) {
     this.routes[path] = data;
     this.trie.add(path, data);
     this.update_last_activity(path);
+    this.statsd.increment('route.add', 1);
```

Comment:
This is causing the failing tests. CHP can be instantiated as a library via js API, in which case the bin script isn't involved.